src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -165,7 +167,8 @@ export class VSCodeNotebookController implements Disposable, IVSCodeNotebookCont
             viewType,
             label,
             this.handleExecution.bind(this),
-            this.getRendererScripts()
+            this.getRendererScripts(),
+            [this.scriptConverter.rootScriptFolder]
```

Comment:
Whoops. This is the change needed for @mjbvz's new API (which hasn't gone through yet)

I'll just comment this out with a note saying this is how we'd use it.